sensors/ev3_uart_sensor_defs.c

```diff
@@ -519,5 +519,327 @@ const struct ev3_uart_sensor_info ev3_uart_sensor_defs[] = {
+				.name		= "ALTITUDE",
+				.data_sets	= 1,
+				.data_type	= LEGO_SENSOR_DATA_S32,
+				.units		= "m",
+				.decimals	= 1,
+			},
+		},
+	},
+	[FATCATLAB_IR] = {
+		/**
+		 * @vendor_name: Fatcatlab
+		 * @vendor_part_name: IR Receiver
+		 * @vendor_website: http://fatcatlab.com/product/ir-receiver
+		 */
+		.name		= FATCATLAB_IR_NAME,
+		.type_id	= FATCATLAB_IR_TYPE_ID,
+		.num_modes	= 1,
+		.num_view_modes	= 1,
+		.mode_info	= {
+			[0] = {
+				/**
+				 * [^ir-mode-values]: Button values:
+				 *
+				 * | Value | Button                 |
+				 * |-------|------------------------|
+				 * | 0     | *none*                 |
+				 * | 1-9   | 1-9 (digits)           |
+				 * | 10    | 0 (zero)               |
+				 * | 11    | + (plus)               |
+				 * | 12    | - (minus)              |
+				 * | 13    | &#x23ee; (previous)    |
+				 * | 14    | &#x23ed; (next)        |
+				 * | 15    | &#x23ef; (play/pause)  |
+				 * | 21    | OK                     |
+				 * | 22    | &#x21b0; (back)        |
+				 * | 30    | &#x23fb; (power)       |
+				 * | 40    | MENU                   |
+				 * | 50    | &#x1f507; (mute)       |
+				 * | 60    | MODE                   |
+				 *
+				 * @description: IR Remote Control
+				 * @value0: Channel 1 (0 to 60)
+				 * @value0_footnote: [^ir-mode-values]
+				 * @units_description: button
+				 */
+				.name		= "IR DATA",
+				.data_sets	= 1,
+				.data_type	= LEGO_SENSOR_DATA_S8,
+			},
+		},
+	},
+	[FATCATLAB_9DOF] = {
+		/**
+		 * @vendor_name: Fatcatlab
+		 * @vendor_part_name: 9DOF Sensor
+		 * @vendor_website: http://fatcatlab.com/product/9dof-sensor
+		 */
+		.name		= FATCATLAB_9DOF_NAME,
+		.type_id	= FATCATLAB_9DOF_TYPE_ID,
+		.num_modes	= 3,
+		.num_view_modes	= 3,
+		.mode_info	= {
+			[0] = {
+				/**
+				 * @description: Gyroscope
+				 * @value0: X-axis rotational speed (-20000  to 20000)
+				 * @value1: Y-axis rotational speed (-20000  to 20000)
+				 * @value2: Z-axis rotational speed (-20000  to 20000)
+				 * @units_description: degrees per second
+				 */
+				.name		= "GYRO",
+				.data_sets	= 3,
+				.data_type	= LEGO_SENSOR_DATA_S32,
+				.units		= "d/s",
+				.decimals	= 1,
+			},
+			[1] = {
+				/**
+				 * @description: Accelerometer
+				 * @value0: X-axis acceleration (-16000 to 16000)
+				 * @value1: Y-axis acceleration (-16000 to 16000)
+				 * @value2: Z-axis acceleration (-16000 to 16000)
+				 * @units_description: standard gravity
+				 */
+				.name		= "ACC",
+				.data_sets	= 3,
+				.data_type	= LEGO_SENSOR_DATA_S32,
+				.units		= "g",
+				.decimals	= 3,
+			},
+			[2] = {
+				/**
+				 * @description: Magnetometer
+				 * @value0: X-axis magnetic flux density (-4800 to 4800)
+				 * @value1: Y-axis magnetic flux density (-4800 to 4800)
+				 * @value2: Z-axis magnetic flux density (-4800 to 4800)
+				 * @units_description: microteslas
+				 */
+				.name		= "MAGNET",
+				.data_sets	= 3,
+				.data_type	= LEGO_SENSOR_DATA_S32,
+				.units		= "uT",
```

Comment:
`u` is used as a substitute for `µ`. Is there a reason the unicode symbol cannot be used?